### ISSUE TITLE
`Transaction` is still a reserved word in 2.4.1

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#3215]
![image](https://user-images.githubusercontent.com/183673/89466386-08adaf80-d774-11ea-9c61-5ea79f6a7e6a.png)

Only applies when `previewFeatures = ["transactionApi"]` is set:

```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {
  provider = "sqlite"
  url      = "file:dev.db"
}

generator client {
  provider = "prisma-client-js"
  previewFeatures = ["transactionApi"]
}

model Role {
  id   Int     @default(autoincrement()) @id
  name String
  users Transaction[]
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

model Transaction {
  id        Int      @default(autoincrement()) @id
  email     String   
  password  String
  name      String?
  roleId    Int
  role      Role    @relation(fields: [roleId], references: [id])
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}
```

**Comment by pantharshit00** [comment]
I can confirm this. Prisma generate also fails:
![image](https://user-images.githubusercontent.com/22195362/89468336-dceef180-d794-11ea-852e-ef46cbd175f1.png)

**Comment by pbelbin** [comment]
Yep, that's the issue I described.

**Comment by Jolg42** [comment]
Thank you for the report! It seems we forgot to remove the reserved keyword from our code.

**Comment by Jolg42** [comment]
The fix will be shipped in next 2.5.0 release next week.
